Frontend.Integration.Tests/IntegrationTestingWebApplicationFactory.cs

```diff
@@ -44,15 +44,16 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
                 config.Sources.Clear();
                 config
                     .AddJsonFile(configPath)
+                    .AddUserSecrets<Program>()
```

Comment:
Wondering what this does?

Reply:
It adds the secret values for the Azure AD integration